suse_caasp

```diff
@@ -431,6 +431,14 @@ if check_rpm docker && [ -e /usr/bin/docker ] && on_admin; then
     plugin_message Done
 
     #############################################################
+    section_header "Getting salt pillars ..."
+    OF=velum-salt-pillars.yml
+    plugin_message "All data stored in $OF"
+
+    docker_exec $OF velum-dashboard entrypoint.sh bundle exec rails runner 'puts(Pillar.all.map(&:parsed_data).to_yaml)'
```

Comment:
It's failing right now: `docker_exec $OF velum-dashboard entrypoint.sh bundle exec rails runner 'puts Pillar.all.pluck(:pillar, :value).to_yaml'` should work.